validate.js

```diff
@@ -0,0 +1,20 @@
+const crypto = require('crypto');
+
+const explorer = 'https://ordinals.com';
+const inscriptionId = 'INSCRIPTION_ID';
+const url = `${explorer}/content/${inscriptionId}`;
+
+const manifestoHash =
+  '1d3c686c4e9a02820bd825700dae97e75a1e157277d0ec2b2f4d851b66f9a5db';
+fetch(url)
```

Comment:
Need to change this with new hash when the new HTML is inscribed.

Reply:
yes, I am currently inscribing. hold my beer. :D